internal/pkg/cli/init_test.go

```diff
@@ -5,682 +5,96 @@ package cli
 
 import (
 	"errors"
-	"fmt"
 	"testing"
 
-	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
 	climocks "github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/mocks"
-	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/manifest"
-	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
-	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
 	"github.com/aws/amazon-ecs-cli-v2/mocks"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 )
 
-func TestInit_Ask(t *testing.T) {
```

Comment:
Note: I remove our old tests as they will be slowly replaced with a mocked actionCommand, I've only added testcases for the codepath that touched "app init".

The next step is to do the same thing "project init" and we should get back test parity over time.